ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py

```diff
@@ -732,6 +732,49 @@ def _squeeze_helper(draw):
     return draw(st.sampled_from(valid_axes))
 
 
+# block
+import hypothesis.strategies as st
```

Comment:
why would you add imports here? Why not add them to the front of the file with other imports?

Reply:
I made the correction sorry for the mistake.